features/bevragen/overzicht.md

```diff
@@ -59,11 +59,24 @@ Een veld wordt gevraagd door het volledig pad van het betreffende veld op te gev
 
 Zie de [fields](./fields.feature) en de [fields fout cases](./fields-fout-cases.feature) feature bestanden voor meer informatie en voorbeelden over het gebruik van veld paden en de fields parameter. 
 
-Voor een Datum en Waardetabel veld wordt altijd alle 'kind' velden geleverd. In de [fields](./fields.feature) zijn onder de volgende rules voorbeelden hiervan opgenomen:
+### Filteren van value type velden
+
+Value types zijn simpele componenten waarvan de velden samen een waarde representeren. De BRP API kent de volgende value types:
+
+- VolledigeDatum
+- DatumOnbekend
+- JaarDatum
+- JaarMaandDatum
+- Waardetabel
+- AdellijkeTitelPredicaatType
+
+Om de waarde in een value type te kunnen interpreteren, zijn alle velden van de value type nodig. Daarom wordt het vragen van één of meerdere velden van een value type behandeld als het vragen van alle velden van de value type. In de [fields](./fields.feature) feature bestand zijn onder de volgende rules voorbeelden hiervan opgenomen:
```

Comment:
Is dit waar? Heb je langFormaat nodig om datum te kunnen interpreteren? Heb je code nodig om omschrijving te kunnen begrijpen?

Reply:
> Is dit waar? Heb je langFormaat nodig om datum te kunnen interpreteren? Heb je code nodig om omschrijving te kunnen begrijpen?

omschrijving heb je meestal wel nodig om code te interpreteren. soort zonder code en/of omschrijving voor adellijke titel/predicaat zegt niets. langFormaat is een extra toevoeging. Alleen langFormaat betekent dat de consumer moet parsen om te kunnen interpreteren om wat voor datum het gaat. JaarMaandDatum zonder type en maand zou ook als een JaarDatum worden geïnterpreteerd. Dit is slechts een voorstel. Als er een betere manier is om dit uit te leggen, dan hoor ik het graag.